packages/components/steps/src/item.vue

```diff
@@ -158,6 +150,15 @@ const space = computed(() => {
   return isSimple.value ? '' : parent.props.space
 })
 
+const containerKls = computed(() => {
```

Comment:
What is the abbreviation for `Kls`? 😕

Reply:
Sorry, I don't know. I'm just following that [Link](https://preview-10754-element-plus.surge.sh/en-US/component/docs.html#class-optimization-in-code).

Reply:
This is a point in document optimization where the class of the template is too long, which is not intuitive to read.

Reply:
Well, I understand that, what I'm not sure is what is the acronym for `Kls`?

Reply:
This is a reference to the previous own demo, which should be homophonic.
`xxClass` => `xxKls`

Reply:
Okay, but it seems that it is not completely unified, and some use `xxClasses`~

Reply:
`xxClasses` should be written in the same way as before the optimization plan was finalized, and in the future, it will be abbreviated as `xxKls`.

Reply:
OK. At that time, I can help with this work.